Examples/src/main/kotlin/com/mapbox/vision/examples/UsbVideoSourceActivityKt.kt

```diff
@@ -0,0 +1,229 @@
+package com.mapbox.vision.examples
+
+import android.graphics.SurfaceTexture
+import android.hardware.usb.UsbDevice
+import android.opengl.GLES11Ext
+import android.opengl.GLES20
+import android.os.Handler
+import android.os.HandlerThread
+import android.widget.Toast
+import com.mapbox.vision.VisionManager
+import com.mapbox.vision.mobile.core.interfaces.VisionEventsListener
+import com.mapbox.vision.mobile.core.models.frame.ImageFormat
+import com.mapbox.vision.mobile.core.models.frame.ImageSize
+import com.mapbox.vision.performance.ModelPerformance
+import com.mapbox.vision.performance.ModelPerformanceConfig
+import com.mapbox.vision.performance.ModelPerformanceMode
+import com.mapbox.vision.performance.ModelPerformanceRate
+import com.mapbox.vision.video.videosource.VideoSource
+import com.mapbox.vision.video.videosource.VideoSourceListener
+import com.serenegiant.usb.USBMonitor
+import com.serenegiant.usb.UVCCamera
+import kotlinx.android.synthetic.main.activity_main.*
+
+/**
+ * Example shows how Vision SDK can work with external USB camera.
+ * [UVCCamera](https://github.com/saki4510t/UVCCamera) library is used to connect to the USB camera itself,
+ * frames from camera are then fed to Vision SDK.
+ */
+class UsbVideoSourceActivityKt : BaseActivity() {
+
+    companion object {
+        private val IMAGE_SIZE = ImageSize(
```

Comment:
It's not possible to get values in runtime here since Vision does not know camera resolution beforehand. Instead, code using Vision should pick some resolution for the camera to listen, and then camera will feed frames with this resolution to Vision, see https://github.com/mapbox/mapbox-vision-android/pull/232/files#diff-7e1d298f0d03dc434e3437877cfc324cR167

Reply:
ah, ok, I've mixed up this idea with frame size like from POI

Reply:
I can leave a comment or rename the variable to make it more clear

Reply:
Renamed to `CAMERA_FRAME_SIZE`